spec/services/outliers/filter_measurements_spec.rb

```diff
@@ -0,0 +1,53 @@
+require "spec_helper"
+
+describe Outliers::FilterMeasurements do
+  it "filters measurement outliers" do
+    max_distance = 1
+
+    calculate_centroid = lambda { |_| [one, one] }
+
+    calculate_distance = lambda do |lng_lat1, lng_lat2|
+      if (lng_lat1 == [one_thousand, one_thousand] || lng_lat2 == [one_thousand, one_thousand])
+        max_distance + 1
+      else
+        max_distance
+      end
+    end
+
+    subject = Outliers::FilterMeasurements.new(max_distance, calculate_centroid, calculate_distance)
+
+    m1 = Measurement.new(
+      longitude: one,
+      latitude: one,
+    )
+
+    m2 = Measurement.new(
+      longitude: one,
+      latitude: one,
+    )
+
+    m3 = Measurement.new(
+      longitude: one_thousand,
+      latitude: one_thousand,
+    )
+
+    measurements = [m1, m2, m3]
+
+
+    actual = subject.call(measurements)
+
+
```

Comment:
no need for new line

Reply:
I do that on purpose. I style tests in the following way
```
# given

# when

# then
```

For example
```
  it "when the points are the same it returns 0" do
     lng_lat = [one, one] # given

     actual = @subject.call(lng_lat, lng_lat) # when

     expect(actual).to eq(zero) # then
  end
```

Unfortunately, sometimes tests get long. In that case, I use two lines instead of one to divide. If the test is long it's a smell though.